index.html

```diff
@@ -20,9 +20,9 @@
         </style>
     </head>
     <body>
-        <h1>Django Stack Installation Guide</h1>
+        <h1>Pytohn/Django Stack Installation Guide</h1>
         <h2>Coauthored by Aggie Wheeler Bateman, Liyun Cecil, Nataliia Carlton</h2>
-        <p>This is a guide that helps you install a Django web application stack on a Google Cloud Services virtual machine. This particular guide includes Ubuntu as the operating system, Apache as the web server, Python/Django as the programming language, and PostgreSQL as the database.</p>
+        <p>This is a guide that helps you install a web application utilized Python/Django stack on a Google Cloud Services virtual machine. This particular stack includes Ubuntu as the operating system, Apache as the web server, Python as the programming language, and PostgreSQL as the database management system.</p>
```

Comment:
Can it say, "web application utilizing a Python/Django stack"? Otherwise, great revisions!